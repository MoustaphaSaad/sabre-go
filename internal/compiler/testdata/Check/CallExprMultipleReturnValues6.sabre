package main

func foo() (int, float32) {
    return 1, 1.5
}

func bar() (int, int, float32) {
    return 1, foo()
}
