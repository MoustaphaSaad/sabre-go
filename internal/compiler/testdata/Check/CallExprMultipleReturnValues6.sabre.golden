>> 	    return 1, foo()
>> 	    ^^^^^^^^^^^^^^^ 
Error[internal/compiler/testdata/Check/CallExprMultipleReturnValues6.sabre:8:5]: expected 3 return values, but found 2
>> 	    return 1, foo()
>> 	    ^^^^^^^^^^^^^^^ 
Note[internal/compiler/testdata/Check/CallExprMultipleReturnValues6.sabre:8:5]: have (int,(int,float32)), want (int,int,float32)
